hardhat.config.ts

```diff
@@ -56,12 +56,12 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
-    mainnet: {
-      url: `https://mainnet.infura.io/v3/${process.env.INFURA_API_KEY}`,
-      accounts,
-      gasPrice: 120 * 1000000000,
-      chainId: 1,
-    },
+    // mainnet: {
```

Comment:
In such cases I suggest using /* */ notation for comments to avoid large unnecessary diffs